lib/api_repo.js

```diff
@@ -429,11 +451,11 @@ function collectProtoDeps(includePath, protoFile, done) {
  * This is a sub task of 'ProtoBased' nodeJS module in _buildProtos, but
  * extracted as a method for the testing.
  *
- * @param {string} outDir - the top of the output directory.
- * @param {[][]string} fileList - the list of the lists of the .proto files
- *   to be copied. Each item is the list of .proto files calculated by
+ * @param {Array.<string[]>} fileLists - the list of the lists of the .proto
```

Comment:
Why mixed notation for array? I would expect either `Array.<Array.<string>>` or `string[][]`